src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -450,6 +450,18 @@ export class DynamicFacet extends Component {
     return allDynamicFacets.indexOf(this) + 1;
   }
 
+  public isCurrentlyDisplayed() {
+    if (!$$(this.element).isVisible()) {
+      return false;
+    }
+
+    if ($$(this.element).hasClass('coveo-active')) {
```

Comment:
Is this condition needed if the default return value is `true`?

Assuming the default condition is correct, then this method can become just:
```
public isCurrentlyDisplayed() {
    return $$(this.element).isVisible();
}
```